scripts/variantstore/wdl/GvsCreateVATFromAnnotations.wdl

```diff
@@ -65,28 +60,12 @@ workflow GvsCreateVATFromAnnotations {
 
 task GetAnnotations {
     input {
-        String? service_account_json_path
         File inputFileofFileNames
     }
-    parameter_meta {
-        inputFileofFileNames: {
-          localization_optional: true
-        }
-    }
-    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
-    String updated_input_files = if (defined(service_account_json_path)) then basename(inputFileofFileNames) else inputFileofFileNames
-
     command <<<
         set -e
 
-        if [ ~{has_service_account_file} = 'true' ]; then
-          gsutil cp ~{service_account_json_path} local.service_account.json
-          export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
-          gcloud auth activate-service-account --key-file=local.service_account.json
-
-          gsutil cp ~{inputFileofFileNames} ~{updated_input_files}
-       fi
-
+      # nothing to do - it is just reading the lines from the input - TODO - is there a better way to do this?
```

Comment:
This was definitely written for the singular use case that is AoU annotation files are in GCP and I want to run them through the python script to transform them into the VAT schema

Reply:
I dont see any reason to have this method at all --and I dont think you need to worry about testing this WDL (maybe we even want to retire it?) since it will be used so rarely and if it is used again it will be in another emergency state that will probably require similar customization

Reply:
Okay, I've removed this file (as Bec mentioned, it's still in the github history).